examples/ping-pong.hs

```diff
@@ -1,15 +1,19 @@
-{-# LANGUAGE OverloadedStrings #-}  -- allows "strings" to be Data.Text
+-- allows "strings" to be Data.Text
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
 
-import Control.Monad (when, forM_, void)
+import Control.Monad (forM_, void, when)
+import Data.Aeson
+import Data.Maybe (fromJust)
 import qualified Data.Text as T
 import qualified Data.Text.IO as TIO
-
-import UnliftIO (liftIO)
-import UnliftIO.Concurrent
-
+import Debug.Trace
 import Discord
-import Discord.Types
+import Discord.Internal.Types.ApplicationCommands
```

Comment:
The point if `Internal` is that the user shouldn't import it. It's a good idea to show how to use the internal stuff, but maybe in a more advanced example. pingpong has enough going on already.

Reply:
Ah, sorry, my code complete must of automatically grabbed from all the way in the internals.